web/src/user_group_popover.ts

```diff
@@ -42,21 +47,26 @@ function get_user_group_popover_items() {
     return $("li:not(.divider):visible a", $popover);
 }
 
-export function handle_keyboard(key) {
+export function handle_keyboard(key: string): void {
     const $items = get_user_group_popover_items();
     popover_menus.popover_items_handle_keyboard(key, $items);
 }
 
 // element is the target element to pop off of;
 // message_id is the message id containing it, which should be selected;
-export function toggle_user_group_info_popover(element, message_id) {
+export function toggle_user_group_info_popover(
+    element: ReferenceElement,
+    message_id: number,
+): void {
     if (is_open()) {
         hide();
         return;
     }
     const $elt = $(element);
-    const user_group_id = Number.parseInt($elt.attr("data-user-group-id"), 10);
-    const group = user_groups.get_user_group_from_id(user_group_id);
+    const user_group_id = $elt.attr("data-user-group-id");
```

Comment:
This might be better called `user_group_id_str` if it's still a string format value.